test/url/url_parse.c

```diff
@@ -218,6 +219,9 @@ void check_query_string(const char *exp, const struct UrlQueryList *act)
 
 void test_url_parse(void)
 {
+  // let's pick a utf-8 locale, since we're also parsing utf-8 text */
+  setlocale(LC_ALL, "en_US.UTF-8");
```

Comment:
As a couple of other tests require this:
- [`test_rfc2047_encode()`
](https://github.com/neomutt/neomutt/blob/master/test/rfc2047/rfc2047_encode.c#L35)
- [`test_rfc2047_decode()`](https://github.com/neomutt/neomutt/blob/master/test/rfc2047/rfc2047_decode.c#L36)

How about adding it to [`test_common()`](https://github.com/neomutt/neomutt/blob/master/test/common.c#L43)

They have a fallback of `C.UTF-8`, which also works for `test_url_parse()`.